types/three/examples/jsm/postprocessing/RenderPixelatedPass.d.ts

```diff
@@ -0,0 +1,28 @@
+import { Scene, Camera, ShaderMaterial, Vector2, MeshNormalMaterial, WebGLRenderTarget } from '../../../src/Three';
+
+import { Pass } from './Pass';
+
+export interface RenderPixelatedPassParameters {
+    normalEdgeStrength?: number;
+    depthEdgeStrength?: number;
+}
+
+export class RenderPixelatedPass extends Pass {
+    constructor(pixelSize: number, scene: Scene, camera: Camera, options?: RenderPixelatedPassParameters);
+    pixelSize: number;
+    resolution: Vector2;
+    renderResolution: Vector2;
+
+    pixelatedMaterial: ShaderMaterial;
+    normalMaterial: MeshNormalMaterial;
+
+    fsQuad: object;
```

Comment:
this is `FullScreenQuad` . It should be accessible from `./Pass`

https://github.com/three-types/three-ts-types/blob/r146/types/three/examples/jsm/postprocessing/Pass.d.ts

Reply:
Should be fixed on [`05af4d0` (#302)](https://github.com/three-types/three-ts-types/pull/302/commits/05af4d022cd093c29d746f91dfa823a376afc96b).
Sorry about that!